packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -110,6 +110,16 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
   }
 
   public focus() {
+    const { options } = this.props;
+    if (options) {
+      for (const option of options) {
+        const elementToFocus = document.getElementById(`${this._id}-${option.key}`);
+        if (elementToFocus && elementToFocus.getAttribute('data-is-focusable') === 'true') {
+          elementToFocus.focus(); // focus on checked or default focusable key
```

Comment:
>focus on checked [](start = 37, length = 16)

Where are you checking if the option is checked? If an option is clicked, are you making the data attribute `data-is-focusable` to true and other options to false?

Also, what is default focusable key?

Reply:
Yes, exactly. Jason made the change here: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx#L88

Reply:
Default focusable key is when none of the options are selected, so it's usually the first option